tests/framework/Samplers/AdaptiveBatch/test_maxv_dp.xml

```diff
@@ -15,7 +14,6 @@
        point (points with more volatility are preferred). This differs from the
        maximum persistence maxima as this looks at distinct local maxima with
        the highest value.
-
```

Comment:
I'm not a big fan of removing the whitespace in XML text. Seems to be an artifact of "prettify"? I'm kind of surprised it's smart enough to find it.